script/Deploy.s.sol

```diff
@@ -395,25 +397,22 @@ contract ArcadiaVaultDeployer is Test {
         vault = new Vault(address(mainRegistry), 1);
         factory.setNewVaultInfo(address(mainRegistry), address(vault), DeployBytes.upgradeRoot1To1, "");
 
-        actionMultiCall = new ActionMultiCall(address(mainRegistry));
+        actionMultiCall = new ActionMultiCall();
         mainRegistry.setAllowedAction(address(actionMultiCall), true);
 
-        ERC20[] memory assets = new ERC20[](2);
-        assets[0] = usdc;
-        assets[1] = weth;
-
         pool_weth =
-        new LendingPool(ERC20(address(weth)), 0x12e463251Bc79677FD980aA6c301d5Fb85101cCb, address(factory), address(0));
-        srTranche_weth = new Tranche(address(pool_weth), "Senior", "SR");
-        jrTranche_weth = new Tranche(address(pool_weth), "Junior", "JR");
+            new LendingPool(ERC20(address(weth)), DeployAddresses.treasury, address(factory), address(liquidator));
+        srTranche_weth = new Tranche(address(pool_weth), "Senior", "s");
+        jrTranche_weth = new Tranche(address(pool_weth), "Junior", "j");
 
         pool_weth.setVaultVersion(1, true);
         pool_weth.setOriginationFee(10);
+        pool_weth.setMaxInitiatorFee(66 * 10 ** 18);
         pool_weth.addTranche(address(srTranche_weth), 50, 0);
         pool_weth.addTranche(address(jrTranche_weth), 40, 20);
         pool_weth.setTreasuryInterestWeight(10);
         pool_weth.setTreasuryLiquidationWeight(80);
```

Comment:
Only give 20% of liquidations to junior tranche? Would bump it in the beginning

Reply:
bumped to 50%